package.json

```diff
@@ -95,32 +95,32 @@
     "@commitlint/config-conventional": "^18.5.0",
     "@documentalist/compiler": "^5.0.0",
     "@jscutlery/semver": "^4.1.0",
-    "@nx/eslint-plugin": "17.1.3",
+    "@nx/eslint": "17.2.8",
+    "@nx/eslint-plugin": "17.2.8",
     "@nx/js": "17.2.8",
-    "@nx/linter": "17.2.8",
     "@nx/react": "17.2.8",
     "@nx/storybook": "17.2.8",
-    "@nx/vite": "17.1.3",
+    "@nx/vite": "17.2.8",
     "@nx/workspace": "17.2.8",
     "@swc/cli": "~0.3.0",
-    "@swc/core": "~1.3.105",
+    "@swc/core": "1.3.105",
     "@testing-library/jest-dom": "^6.2.0",
     "@testing-library/react": "14.1.2",
     "@testing-library/user-event": "^14.5.2",
     "@types/classnames": "^2.3.1",
     "@types/node": "^20.11.4",
     "@types/rc-tooltip": "3.7.10",
-    "@types/react": "^18.2.38",
-    "@types/react-dom": "^18.2.18",
-    "@typescript-eslint/eslint-plugin": "^6.19.1",
-    "@typescript-eslint/parser": "^6.19.0",
-    "@vitejs/plugin-react": "~4.2.0",
-    "@vitest/coverage-v8": "^0.34.6",
-    "@vitest/ui": "^0.34.7",
+    "@types/react": "18.2.24",
```

Comment:
This was downgraded. Same for `@types/react-dom` and `eslint`. Is that intentional?

Reply:
Yes, NX is compatible with certain versions, and thus running a NX migration might downgrade. I think this is mostly an issue with the build toolchain (Vite etc.), so maybe we shouldn't downgrade React. We probably also need to exclude Vite from Dependabot. I will try to update the PR